tendrl/node_agent/persistence/disk.py

```diff
@@ -0,0 +1,69 @@
+import json
+import time
+
+
+class Disk(object):
```

Comment:
I understand the logic behind this class not being extended from etcdobject and rather a json representation of the same would be written one shot to etcd :)